web/Migrations/20220726082722_PropertyContainsUserWhoRented.cs

```diff
@@ -0,0 +1,188 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace BricksAndHearts.Migrations
+{
+    public partial class PropertyContainsUserWhoRented : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<string>(
+                name: "TownOrCity",
+                table: "Property",
+                type: "nvarchar(max)",
+                nullable: true,
+                oldClrType: typeof(string),
+                oldType: "nvarchar(max)");
```

Comment:
I don't think all of these should be here, maybe need to regenerate this after rebasing + applying migrations?